config/project-scratch-def.json

```diff
@@ -0,0 +1,13 @@
+{
+  "orgName": "Subscription Management",
+  "edition": "Developer",
+  "features": ["EnableSetPasswordInApi"],
```

Comment:
Is there no way to create a SM org using features? Shouldn't customers be able to create scratch orgs using SM?